qdax/core/containers/mels_repertoire.py

```diff
@@ -0,0 +1,311 @@
+"""This file contains the class to define the repertoire used to
+store individuals in the Multi-Objective MAP-Elites algorithm as
+well as several variants."""
+
+from __future__ import annotations
+
+from typing import Callable, Optional
+
+import jax
+import jax.numpy as jnp
+from jax.flatten_util import ravel_pytree
+
+from qdax.core.containers.mapelites_repertoire import (
+    MapElitesRepertoire,
+    get_cells_indices,
+)
+from qdax.types import Centroid, Descriptor, ExtraScores, Fitness, Genotype, Spread
+
+
+def _dispersion(descriptors: jnp.ndarray) -> jnp.ndarray:
```

Comment:
Minor suggestion: perhaps these functions belong elsewhere in a utils/ file ? I feel like they can be used by other methods as scores or metrics

Reply:
I think if it becomes something used in multiple places, perhaps it can then be refactored? Since it is a private function, it seems it can be moved around pretty easily.